graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/CodeGen.kt

```diff
@@ -156,9 +160,29 @@ class CodeGen(private val config: CodeGenConfig) {
                 }
             }
         }
+
         return document
     }
 
+    private fun loadTypeMappingsFromDependencies() {
+        // process type mappings from dependencies
+        config.schemaJarFilesFromDependencies.forEach { file ->
+            val typeMappingFile = JarFile(file).getJarEntry("META-INF/dgs.codegen.typemappings")
```

Comment:
I think we should close the JarFile once we're done with it, so I'd probably do something like:
```kotlin
JarFile(file).use { jarFile ->
 // ...
}
```